### PR TITLE
fix adoc syntax in SLL section

### PR DESCRIPTION
**Pull request by keichwa**
partial backport of #3083 (per feedback)

- 4.3.13

### CODE REVIEW

**Comment by keichwa** [comment]
4.3.13 backport probably req.

File: modules/specialized-guides/pages/salt/salt-formula-liberate.adoc [hunk 1]
```diff
@@ -278,7 +278,7 @@ This method makes use of the bootstrap script created above for onboarding new s
 
 . In the {productname} {webui}, open the System Details page of any registered client you want to migrate to {sll}.
 
-. Click the [guimenu]`Reactivation`` tab.
+. Click the [guimenu]``Reactivation`` tab.
```
Comment (nodeg) [inline modules/specialized-guides/pages/salt/salt-formula-liberate.adoc:281]:
```suggestion
. Click the [guimenu]`Reactivation` tab.
```

**Review by nodeg — CHANGES_REQUESTED** [review]

**Comment by nodeg** [comment]
And please also add https://github.com/uyuni-project/uyuni-docs/pull/3083/files#diff-abd981ee5cd3e5fa04b6b46a701f28016af16d01c97ca5b6a67505977edada1eL286-R173 which is a typo fix.

**Comment by nodeg** [comment]
One more thing:
image

File: modules/specialized-guides/pages/salt/salt-formula-liberate.adoc (hunk 1 above)
Comment (keichwa) [reply modules/specialized-guides/pages/salt/salt-formula-liberate.adoc:281]: I think we still want to stick with double backtics here.

**Comment by keichwa** [comment]
> One more thing:

strange artifact. ATM, I cannot spot the cause for it.  Building docs...

**Review by nodeg — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Pull request merged**